technical_standards/testing/strategy.md

```diff
@@ -0,0 +1,96 @@
+# Strategy
+
+The testing strategy is one of the key aspects to consider when testing as this what will help us to focus on what matters to us.
+
+
+## WP Media testing strategy
+
+As we are a small team, your time is valuable and limited, so we cannot afford to spend this precious resource on automating fragile tests that will end up costing us more than what we invested.
+
+### We build Diamonds not pyramids
+
+To avoid fragile tests are using a testing strategy based mainly on integration tests and try to rely as little as possible on unit tests.
+
+This makes our testing pyramid look more like a diamond as integration tests are more numerous than your unit tests.
+
+The choice of integration tests over unit ones is related to multiple reasons.
+
+First, they are black box tests or in other words, they do not rely on the implementation of the application unlike unit tests which are close to the implementation.
+
+This allows tests to be anti-fragile and ease the amount of maintenance required by our tests.
+
+Second, we value tests as a way to communicate by using integration tests primary instead of unit tests, we are putting business requirements at the center of our testing process allowing communication with the product team and QA.
+
+![](./_images/diamont.png)
```

Comment:
@MathieuLamiot @CrochetFeve0251 Was this discussed anywhere? This has never been an official guideline for our team